CHANGES

```diff
@@ -1,3 +1,72 @@
+Release 1.10 (development)
+--------------------------
+
+.. rubric:: Enhancements
+
+* Print sizes in ``rauc info`` also in human-readable form.
+* Add FTPS support for bundle download (only for use with casync, not for
+  streaming). (by Christian Meusel)
+* Improve progress granularity to provide more realistic weighting of substeps.
+* Add fine-grained progress updates during image copying and archive
+  extraction. (based on work by Lars Poeschel)
+* Return manifest meta data in ``rauc info`` and via the InspectBundle D-Bus
+  method.
+* Add new 'json-2' output format for ``rauc info`` that matches the
+  InspectBundle D-Bus method structure.
+* Improve error message for failed boot slot detection.
+* Allow exFAT as a local filesystem for plain bundles. (by Stefan Wahren)
+* Add optional pre-check for verity bundles. (by Christian Hitz)
+  This is useful if the same bundle needs to be transferred and installed to
+  multiple systems in sync.
+* Add support for custom variables in the system-info handler and pass them to
+  other handlers.
+* Show a warning during bundle creation if no format is specified in the
+  manifest.
+  This should hopefully encourage migration to the verity format.
+* Introduce an installation transaction UUID, which is stored in the slot
+  status.
+  This can be used to infer which slots have been updated by the same
+  transaction.
+  In a future release, this will be useful to correlate logs messages.
+* Use a shorter connect timeout for streaming to avoid waiting for 25 minutes.
+
+.. rubric:: Bug fixes
```

Comment:
Maybe worth to add:

* Fix unintentional forwarding of full custom handler args (defined in manifest.conf) to the system.conf-defined handlers

Just in case someone stumbles across having used this undocumented behavior...

Also worth being noted:

* Re-add missing `--key` argument for providing the decryption key to help and man page